BLSQ/iaso
# IA-2613: Use Org unit change requests old values while approved

Using old field values while change request is approved, following [those](https://github.com/BLSQ/iaso/pull/1047) changes

Related JIRA tickets : IA-2613
## Self proofreading checklist


## Changes

- refactor `useNewFields` for readability
- review status colors 
- using old values if status is approved

## How to test

Remove old change requests you did before Marc previous PR.
Create 3 change requests
Reject, approved and let the last one to new (use the dashboard, not the admin)
Make sure the dialog renders properly the fields with correct colors, also if this is the approved one, values on the left should be different than the modified version.


## Print screen / video

<img width="1620" alt="Screenshot 2024-01-24 at 10 35 30" src="https://github.com/BLSQ/iaso/assets/12494624/fcb0fed2-083f-4bde-9297-0f5a38b8dc2b">
<img width="1624" alt="Screenshot 2024-01-24 at 10 35 20" src="https://github.com/BLSQ/iaso/assets/12494624/694a5727-5735-4a2e-9be1-04aaf492f6d3">
<img width="1592" alt="Screenshot 2024-01-24 at 10 35 11" src="https://github.com/BLSQ/iaso/assets/12494624/eb64acbe-b264-471a-9443-869b0710f479">
